Fix classnames for block nesting

joemcgill opened the pull request:
This adds support for the layout attribute to be passed to the render callback so the `InnerBlocks` component can update the classnames value of the maps block when it's included in a nested layout and also uses `classnames()` to simplify the way classNames are set inside the editor.

joemcgill committed:
Simplify conditional classnames

This uses conditional class setting functionality in `classnames()` to simplify the setting of classes.

joemcgill committed:
Rebuild assets and update hashes

joemcgill committed:
Fix whitespace inside brackets

kadamwhite reviewed (approved):
Nice cleanup. I don’t see any code relating to this, though:

> adds support for the layout attribute to be passed to the render callback

Doesn’t look like any attributes are affected here...?

joemcgill committed:
Add support for the layout attribute in the maps render callback

This adds the layout attribute to the maps render callback so the attribute can be passed from the `InnerBlocks.Content` component to the maps block when it's included inside of a columns block.

kadamwhite reviewed (approved):
New commit looks good

joemcgill committed:
Merge upstream changes and update build/hashes
